src/explore-education-statistics-frontend/src/modules/methodologies/__tests__/MethodologyIndexPage.test.tsx

```diff
@@ -101,6 +101,41 @@ const testThemes: MethodologyTheme[] = [
       },
     ],
   },
+  {
+    id: 'theme-3',
+    summary: 'Theme 3 summary',
+    title: 'Theme 3',
+    topics: [
```

Comment:
Personally, I probably would have leaned towards doing this in the backend by removing topics as we're in the area anyway.